package.json

```diff
@@ -10,8 +10,8 @@
     "test": "jest --coverage",
     "snyk": "snyk test",
     "clean": "rimraf ./dist && mkdirp ./dist ./dist/js ./dist/css ./dist/fonts ./dist/img",
-    "js": "rimraf ./dist/js/app.min.js.map && browserify -d -g [ envify --NODE_ENV ${NODE_ENV:-production} ] -t [ babelify --presets [ es2015 react ] --plugins [ babel-plugin-transform-object-rest-spread ] ] -p [ minifyify --map app.min.js.map --output ./dist/js/app.min.js.map --uglify [ -cm ] ] ./src/js/index.js | bundle-collapser - > ./dist/js/app.min.js",
-    "js:dev": "rimraf ./dist/js/app.min.js.map && browserify -d -t [ babelify --presets [ es2015 react ] --plugins [ babel-plugin-transform-object-rest-spread ] ] -e ./src/js/index.js -o ./dist/js/app.min.js",
+    "js": "rimraf ./dist/js/app.min.js.map && rimraf ./dist/js/app.min.js.bak && browserify -d -g [ envify --NODE_ENV ${NODE_ENV:-production} ] -t [ babelify --presets [ es2015 react ] --plugins [ babel-plugin-transform-object-rest-spread ] ] -p [ minifyify --map app.min.js.map --output ./dist/js/app.min.js.map --uglify [ -cm ] ] ./src/js/index.js | bundle-collapser - > ./dist/js/app.min.js",
+    "js:dev": "rimraf ./dist/js/app.min.js.map && rimraf ./dist/js/app.min.js.bak && browserify -d -t [ babelify --presets [ es2015 react ] --plugins [ babel-plugin-transform-object-rest-spread ] ] -e ./src/js/index.js -o ./dist/js/app.min.js",
```

Comment:
🚀  had noticed this one too!

Reply:
Actually, `js:dev` still doesn't build right. You'll see things like `##KEYCLOAK_URL##/` in the console after running `js:dev`. I think _maybe_ it needs `env` run too?

Reply:
Fixed!